<launch>
    <node pkg="capability_map_generator" type="capability_map_generator" name="capability_map_generator" args="-p $(find capability_map_generator)/maps/dummy_map.cpm -n 200 -r 0.1 -x 0.0 -x 2.5 -y 0.05 -y 0.05 -z 0.45 -z 0.45" output="screen">
        <param name="reachability_interface" value="capability_map_generator::ReachabilityDummyInterface"/>
        <param name="group_name" value="not_needed" />
        <param name="base_name" value="torso_lift_link" />
        <param name="tip_name" value="no_tip_link" />
    </node>
</launch>

